src/ios/Flutter/AppFrameworkInfo.plist

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+  <key>CFBundleDevelopmentRegion</key>
+  <string>en</string>
+  <key>CFBundleExecutable</key>
+  <string>App</string>
+  <key>CFBundleIdentifier</key>
+  <string>ch.zosimovy.projects.$(ZSTUDIO_APP_SUFFIX)</string>
```

Comment:
`ch.zosimovy.projects.zp$(SOFTWINE_APP_SUFFIX)`, check and change where required